runner_manager/main.py

```diff
@@ -21,3 +23,14 @@ def startup_event():
 @app.get("/_health")
 def health():
     return Response(status_code=200)
+
+
+@app.post("/webhook", status_code=202)
+async def webhook(request: Request):
```

Comment:
We should be able to defined `WebhookEvent` directly, fastapi supports pydantic models to be defined as parameter in the function and will then automatically check if the request is valid as well as producing the proper openapi spec/docs.
```suggestion
async def webhook(webhook: WebhookEvent):
```